gobblin-runtime/src/main/java/org/apache/gobblin/runtime/task/TaskIFaceWrapper.java

```diff
@@ -75,6 +78,8 @@ public String getProgress() {
   public void run() {
     try {
       this.underlyingTask.run();
+    } catch (Exception e) {
```

Comment:
For compaction job, we already have logs in MRTask. Maybe we should propagate the exception back to GobblinMultiTaskAttempt.

Reply:
Removed.